integrations/sonarqube/.port/resources/port-app-config.yaml

```diff
@@ -30,15 +30,15 @@ resources:
       entity:
         mappings:
           blueprint: '"sonarQubeAnalysis"'
-          identifier: .analysisId
-          title: .__commit.message // .analysisId
+          identifier: .analysisId // .__project + "-" + .key
+          title: .__commit.message // .analysisId // .title
           properties:
-            branch: .__branchName
+            branch: .branch // .__branchName
             fixedIssues: .measures.violations_fixed
-            newIssues: .measures.violations_added
-            coverage: .measures.coverage_change
-            duplications: .measures.duplicated_lines_density_change
-            createdAt: .__analysisDate
+            newIssues: .measures.violations_added // .__measures[]? | select(.metric == "new_violations") | .period.value
+            coverage: .measures.coverage_change // .__measures[]? | select(.metric == "new_coverage") | .period.value
+            duplications: .measures.duplicated_lines_density_change // .__measures[]? | select(.metric == "new_duplicated_lines_density") | .period.value
+            createdAt: .analysisDate // .__analysisDate
```

Comment:
I am unable to follow the convention you used, for some properties you added some mapping to be first and some as last.

1. Is that on purpose, if so we need to document it somehow.
2. seems that most of the properties have different mapping resources, so maybe we should just define a new "mapping" resource and we will use the relevant one per whether its onpremise or not. 
@yairsimantov20  would like to get your input on it.

Reply:
Yes, the reason why we're having different resource mapping is because of the enhancements: One API route is unable to contain all the needed information so I have to call other APIs to get the full data required.

I like the idea of having another mapping for the same `analysis` kind but filter when it's on-premise. Challenge will be the best way to determine whether it is on-premise or not

Reply:
sure understood regarding the different APIs to get the same data, what I referred to is the order, which makes it harder after committed to understand whether which "mapping" is on-premise and which is saas.

For example, in branch key, you added the mapping as the first executed while in newIssues you added as the second option.

I think the way to understand whether its on-premise is using the `sonarIsOnPremise` parameter that you just added to the config. 
And have it as two different kinds instead of the same one, that way when you in on-premise you ignore the saas one and the same for the opposite